### DISCUSSION TITLE
Does pypdf reads text of other language than english?

### DISCUSSION THREAD

**Discussion by mahimairaja** [py-pdf/pypdf#2347] [Q&A] [answered]
Can it read - Arabic?

**Comment by pubpub-zz** [comment, marked as answer]
if you mean extract text, the function should be compatible with most of the character sets(arabic, asian).  It if based on font definition, and some character set may not work

**Reply by MartinThoma** [reply]
In short: "Yes, pypdf can extract non-English texts" :smile: :+1:
